src/open_inwoner/plans/models.py

```diff
@@ -115,6 +116,15 @@ class Plan(models.Model):
         blank=True,
         help_text=_("How do you intend to achieve this goal?"),
     )
+    note = ckeditor_fields.RichTextField(
+        max_length=1000,
```

Comment:
Leave the `max_length` out. Since this is a single note composed of several logically distinct notes, it's not unlikely that we're going to hit the 1000 mark very soon.